Source/CarthageKit/Xcode.swift

```diff
@@ -950,13 +950,29 @@ public func buildScheme(scheme: String, withConfiguration configuration: String,
 	}
 
 	return BuildSettings.SDKsForScheme(scheme, inProject: project)
+		.flatMap(.Concat) { sdk -> SignalProducer<SDK, CarthageError> in
+			var argsForLoading = buildArgs
+			argsForLoading.sdk = sdk
+
+			return BuildSettings
+				.loadWithArguments(argsForLoading)
+				.filter { settings in
+					// Filter out SDKs that require bitcode when bitcode is disabled in
+					// project settings. This is necessary for testing frameworks, which
+					// must add a User-Defined setting of ENABLE_BITCODE=NO.
+					return settings.bitcodeEnabled.value == true || ![.tvOS, .watchOS].contains(sdk)
+				}
+				.map { _ in sdk }
+		}
 		.reduce([:]) { (sdksByPlatform: [Platform: [SDK]], sdk: SDK) in
```

Comment:
Could we maybe change `sdksByPlatform` to `[Platform: Set<SDK>]` to filter out duplicates?